### PR TITLE
#162685656 Integrate Coveralls code coverage service with badge

### PR DESCRIPTION
**Pull request by tomuhenry**
**What does this PR do?**
This PR delivers a chore that integrates Coveralls code coverage service

**Description of Task to be completed?**
Connecting and configuring _Coveralls_ with _GitHub_ and _Travis_ to have code test coverage after successful Travis build.

**What are the relevant pivotal tracker stories?**
[#162685649](https://www.pivotaltracker.com/story/show/162685656)

[Link to coveralls](https://coveralls.io/github/andela/Ah-backend-guardians)

**Screenshot:**
screen shot 2019-01-08 at 7 11 06 pm

### CODE REVIEW

**Review by fahadmak — DISMISSED** [review]

**Review by BarnaTB — CHANGES_REQUESTED** [review]
Your PR has too many commits so you could adopt of either squashing your commits or amending them to previous ones.
Also looking at your `README`, your badge shows `unknown` as opposed to what is shown on the coveralls website itself so you might want to put the badge that maps to your exact branch since it looks like it is displaying the `develop` branch.

**Review by fahadmak — COMMENTED** [review]
Try to squash the commits into one commit

**Comment by tomuhenry** [comment]
I've done a rebase and merged the commits into one @fahadmak 
I have also made the change to the **readme** and added the badge for the right branch @BarnaTB

File: .coveralls.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+repo_token: 6ggWHxHTG2JvepQ98oe4baMzh7EWPOtDM
```
Comment (archbishop12) [inline .coveralls.yml]: @tomuhenry Could you add this as an environment variable. Thanks.

Comment (tomuhenry) [reply .coveralls.yml]: Ok... Working on it

File: .travis.yml [hunk 2]
```diff
@@ -4,6 +4,9 @@ python:
 addons:
   postgresql: "9.4"
 
+env:
+    - repo_token: 6ggWHxHTG2JvepQ98oe4baMzh7EWPOtDM
```
Comment (archbishop12) [inline .travis.yml]: @tomuhenry Add it to the environment variables on travis, it should not be visible here. Thanks

**Comment by fahadmak** [comment]
LGTM

**Comment by VivianDoreen** [comment]
LGTO Great work @tomuhenry

**Review by myrdstom — APPROVED** [review]

**Review by BarnaTB — APPROVED** [review]
Great work Tom! LGTM.

**Review by archbishop12 — APPROVED** [review]

**Pull request merged**